frontend/src/pages/LogsPage/SearchForm/SearchForm.tsx

```diff
@@ -148,7 +145,7 @@ const SearchForm = ({
 					{!hideDatePicker && (
 						<PreviousDateRangePicker
 							emphasis="low"
-							selectedDates={selectedDates}
+							selectedDates={[startDate, endDate]}
```

Comment:
Think this is a pre-exisiting issue, but may be more obvious with the changes to fetch newer logs. The time frame gets updated from "Last X" to a custom time frame 

https://github.com/highlight/highlight/assets/17744174/51dda78b-ca1d-422b-a176-e6cbc7761b0e

Reply:
fixed in b2c888243c11c97daa64e53fe06959eb21343b70